app/assets/javascripts/application.js

```diff
@@ -432,7 +432,7 @@ function setPowerState(item, status){
       $('#loading_power_state').remove();
     }
   }else{
-    $('#loading_power_state').text(_('Unknown power state'))
+    $('#loading_power_state').text(__('Unknown power state'))
```

Comment:
$('#loading_power_state') should be cached when the fix is made. No need to repeat DOM operation.